Makefile

```diff
@@ -0,0 +1,19 @@
+CODE_DIRS = monitoring/monitoring_service/ monitoring/request_collector/ pathfinding/pathfinding_service
```

Comment:
What about `libs`?

Reply:
Good point

Reply:
Oh, there's quite some errors with that. I'll create a follow-up issue.